source/includes/fact-connect-intro.rst

```diff
@@ -0,0 +1,7 @@
+The :ref:`mongosync <c2c-mongosync>` utility creates a connection
+between MongoDB clusters. ``mongosync`` can make a connection between:
+
+- :ref:`self-managed clusters <server-replica-set-deploy-all>`,
+- :atlas:`MongoDB Atlas </>` hosted clusters, or 
```

Comment:
same as above, is it correct to end the bullet with ", or"

Reply:
Yes, as above